### ISSUE TITLE
Frontend: edit-buttons on digital identities-overview page

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#71]
## User story
1. As a {HR employee}
2. I need {an edit button for each digital identity}
3. So that {I can make changes to the existing digital identity}

## Acceptance criteria
* "edit"-button in each row of the digital-identity-overview page on the right side of each digital identity
* button links to a page where the changes can be made later (only dummy page necessary at this step)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
